scripts/stellar_cli.mjs

```diff
@@ -4,12 +4,18 @@ import { execSync } from "child_process";
 
 const repoUrl = "https://github.com/stellar/stellar-cli.git";
 const localRepoPath = "./stellar-cli-repo";
```

Comment:
```suggestion
```

Reply:
We are still getting latest version using git...

Reply:
we can hit github's api instead, which I _think_ makes more sense if we're not using any of the repo.

```console
$ http https://api.github.com/repos/stellar/stellar-cli/tags | jq --raw-output '.[0].name'
v22.6.0
```

Reply:
Yeah we can do that instead 👍🏼